meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jsx

```diff
@@ -21,6 +22,40 @@ export const BudgetingProposalListItem = (props) => {
         safeLocale
       )}`
 
+  // Before passing this data to 'ListItemBadges' component
+  // it needs to be normalized as the data types are different
+  // category: null || object
+  // labels: [] || array
+  // budget: 0 || number
+  // pointLabel: '' || string
+  // modFeedback: [null, null] || array
```

Comment:
Uh. This makes sense, but shouldn't we change the serializer to give better values? We will have to change this for TS anyway. And in a lot of places. Maybe here is a good place to start and find out how we would like to get data from the API that can be there or not? We have all the examples you can think of in one place. :p

Reply:
@fuzzylogic2000 that is true. I will look into how to send them similar from the serializer